client/src/Context/PostsContext.tsx

```diff
@@ -5,7 +5,7 @@ import PostInfo from '../Interfaces/post/IAddPost';
 import ITag from '../Interfaces/post/ITag';
 
 interface IStore{
-  filterObj?:PostInfo;
+  filterObj?:Omit<PostInfo, 'UserId'>;
   setFilterObj?:Function;
```

Comment:
no need to call them filterObj and setFilterObj, just call them filter and setFilter

Reply:
@mohmmed23

Reply:
I named it this name because it contains more than one filter, I wanted the name to express the content